exo-add/src/cli.ls

```diff
@@ -60,6 +62,19 @@ function help
   console.log help-message
 
 
+function check-for-service {service-role, existing-services}
+  if existing-services.includes service-role
+    console.log red "Service '#{service-role}' already exists in this application"
```

Comment:
Please match the existing formatting: remove the quotes and make the service role cyan